rest-api-spec/src/main/resources/rest-api-spec/api/nodes.reload_secure_settings.json

```
@@ -27,11 +27,15 @@
         }
       ]
     },
-    "params":{
-      "timeout":{
-        "type":"time",
-        "description":"Explicit operation timeout"
+    "params": {
+      "timeout": {
+        "type": "time",
+        "description": "Explicit operation timeout"
       }
+    },
+    "body": {
+      "description": "The password for the elasticsearch keystore",
```

This implies that the body should be the raw password, I think we want something more like:
```suggestion
      "description": "An object containing password for the elasticsearch keystore",
```